activerecord/lib/active_record/fixtures.rb

```diff
@@ -4,6 +4,7 @@
 require "yaml"
 require "zlib"
 require "set"
+require "global_id/uri/gid"
```

Comment:
`URI::GID` is used below, but activerecord does not depend explicitly on `globalid`. Maybe it should?

Reply:
It should not. Are you calling the method that uses it? That method should not even exist in Active Record. It was added to support Action Text attachment in fixtures.

Reply:
ok. should I drop this require from the PR then?

Reply:
Yes, please. I'm fixing this in another way though, so the usage of gid will not be here anymore (I'm extracting this method to the global_id gem).

Reply:
done